.gitleaks.toml

```diff
@@ -1,5 +1,5 @@
 [allowlist]
-  description = "Our test install exports a test only MINIO ACCESS KEY"
-  paths = [
+  files = [
```

Comment:
Needs to stay "paths" - "files" is for specific-filenames (and is going to be deprecated soon anyway)